### DISCUSSION TITLE
Commands without bot username

### DISCUSSION THREAD

**Discussion by notRyuk** [python-telegram-bot/python-telegram-bot#2534] [Q&A] [answered]
As I'm testing my bot I found out that in *groups* I have to type `/mycommand@mybotusername`. But I want to respond even for `/mycommand` . 
How to do that?

**Comment by Eldinnie** [comment, marked as answer]
You don't.
This is determined by the api for which messages your bot will receive.
https://core.telegram.org/bots/faq#what-messages-will-my-bot-get

**Reply by Bibo-Joshi** [reply]
To be precise, after turning off privacy mode, the bot can react to `/mycommand` 😉
